src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/AttachExceptionRecordToExistingCaseTest.java

```diff
@@ -195,6 +192,22 @@ private void verifyExistingCaseIsUpdatedWithExceptionRecordData(
             exceptionRecord.getId(),
             expectedExceptionRecordsSize
         );
+
+        assertCorrectOcrDataAfterUpdate(originalCase, exceptionRecord, updatedCase);
```

Comment:
Ocr data is always present as part of exception record creation for new application. Whenever a case is updated we should never update/add ocr data. So I am not sure why we have this assertion ?

Reply:
Ok, I assumed that exception record can change OCR data. Will remove this case.

Does it mean OCR data will only be included when classification is NEW_APPLICATON?

Reply:
yes that's correct it will only when classification is NEW_APPLICATON or EXCEPTION(because this case might be any)